tools/pkglint/package.json

```diff
@@ -16,11 +16,11 @@
     "pkglint": "bin/pkglint"
   },
   "scripts": {
-    "build": "tsc -b && tslint -p . && chmod +x bin/pkglint",
+    "build": "tsc -b && tslint -p . && eslint . --ext .ts,.js --ignore-path .gitignore && chmod +x bin/pkglint",
```

Comment:
Do we need to actually have both `.ts` and `.js` processed?

- All the `js` (maybe except some rare exceptions) is generated from the `ts`
- Doe this also process `.d.ts` then? Maybe that's undesirable, too?

Reply:
Generated `.js,.d.ts` files are `.gitignore`d, and I use ` --ignore-path .gitignore`, so it should be a non-issue.

Reply:
I see. Makes sense.